### PR TITLE
Use README as package description

### PR DESCRIPTION
**Pull request by alexshoe**
- Reads the contents of `README.md` as `long_description`, to be displayed as the PyPI project description
- Updates the "Getting Started" instructions for the README
- Moves contribution steps from README to CONTRIBUTING

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -15,6 +16,6 @@
     packages=[package_name],
     include_package_data=True,
     license=package["license"],
-    description=package["description"] if "description" in package else package_name,
```
Comment (alexcjohnson) [inline setup.py:18]: judging from the `setup.py` for Dash, we should [leave `description` in place](https://github.com/plotly/dash/blob/a9eb3434023880c229882dd2aacb7d501e7eb4d2/setup.py#L22) when we add `long_description`, as well as adding [`long_description_content_type`](https://github.com/plotly/dash/blob/a9eb3434023880c229882dd2aacb7d501e7eb4d2/setup.py#L27)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,4 +1,20 @@
 # CONTRIBUTING
 
-This project was generated by the [dash-component-boilerplate](https://github.com/plotly/dash-component-boilerplate) it contains the minimal set of code required to create your own custom Dash component.
+## Build manually
+
+To generate builds manually, run the following:
+
+```sh
+$ npm install webpack webpack-cli
```
Comment (alexcjohnson) [inline CONTRIBUTING.md:8]:
```suggestion
$ pip install -r requirements-dev.txt
$ npm install
```

File: setup.py (hunk 1 above)
Comment (alexshoe) [reply setup.py:18]: Done!

**Comment by alexcjohnson** [comment]
While we're at it, let's add a few more things to `setup.py` (inside `setup()`), perhaps:
```
    python_requires=">=3.6",
    url="https://github.com/plotly/dash-player",
    install_requires=["dash>=1.6.1"],
    classifiers=[
        "Programming Language :: Python :: 3",
        "Framework :: Dash",
        "License :: OSI Approved :: MIT License",
        "Operating System :: OS Independent",
    ],
```

**Review by alexcjohnson — APPROVED** [review]
💃

**Pull request merged**